python/lsst/ts/scheduler/driver.py

```diff
@@ -10,6 +13,80 @@
 __all__ = ["Driver", "DriverParameters"]
 
 
+class DriverTarget(Target):
+    def __init__(self, targetid=0, fieldid=0, band_filter="",
+                 ra_rad=0.0, dec_rad=0.0, ang_rad=0.0,
+                 num_exp=0, exp_times=[]):
+        """Initialize the class.
```

Comment:
Thanks for the doc. But...:
- Please document what DriverTarget is for (in a doc string for the class)
- Please consider moving this doc string to the class doc string, as per numpydoc rules and https://developer.lsst.io/python/numpydoc.html . This is admittedly for DM, but I hope T&S will adopt it.